content/08.transmission.md

```diff
@@ -1,24 +1,68 @@
+Taken together, these results indicate that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887].
+However, viral shedding is not necessarily indicative of contagiousness, and this study also concluded that the risk of spreading the infection was low after ten days from the onset of symptoms, as they determined the viral load in sputum was unlikely to pose a significant risk based on their efforts to culture samples _in vitro_ [@doi:10.1038/s41586-020-2196-x].
+
+The extent to which asymptomatic (or presymptomatic) individuals are able to transmit SARS-CoV-2 has been a question of significant interest in the first six months of the pandemic.
+Early reports (February and March 2020) emerged describing transmission from presymptomatic SARS-CoV-2-positive individuals to close family contacts [@doi:10.1001/jama.2020.2565; @doi:10.1016/j.ijid.2020.03.027].
+One of these reports [@doi:10.1016/j.ijid.2020.03.027] also included a description of an individual who tested positive for SARS-CoV-2 but never developed symptoms.
+Later analyses have sought to estimate the proportion of infections that could be traced back to a presymptomatic or asymptomatic individual.
+Estimates of the proportion of individuals with asymptomatic infections have varied widely.
+The proportion of asymptomatic individuals on board the Diamond Princess cruise ship, which was the site of an early COVID-19 outbreak, was estimated at 17.9% [@doi:10.2807/1560-7917], 
+In contrast, a model using the prevalence of antibodies among residents of Wuhan, China estimated a much higher of asymptomatic cases, at approximately 7 in 8, or 87.5% [@doi:10.1093/pcmedi/pbaa032/5901532].
+An estimate of the population of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% [@doi:10.1016/j.eclinm.2020.100533]. 
+The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms [@doi:10.1038/s41591-020-0965-6].
+As a result, the potential for individuals who do not know they have COVID-19 to spread the virus is of significant concern.
+In the two cities studied to estimate incubation period, an estimated 40-50% and 60-80%, respectively, were estimated to be caused by contact with an asymptomatic individuals [@doi:10.7554/eLife.57149].
+An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].
+They argued that the town's lock down was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
```

Comment:
```suggestion
They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
```